packages/preinstall-always-fail/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@lavamoat/preinstall-always-fail",
   "version": "2.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "package.json",
```

Comment:
there is no `index.js` in this package, but there is a `package.json`. 😄

Reply:
would agree on omitting based on
> if your package is named `foo`, and a user installs it, and then does `require("foo")`, then your main module's exports object will be returned.

since `package.json` doesn't export an obj, but
> If `main` is not set, it defaults to `index.js` in the package's root folder.

which is dubious since `index.js` doesn't exist

so guess it's one of those pick your poison decisions

Reply:
`require('some.json')` just gives you an object, so `require('preinstall-always-fail')` will now give you an object instead of throwing an exception.  It doesn't really matter, given that this package isn't intended to do anything except _be_.